### PR TITLE
fix(communication): display changes to ConnectedAPs in SubNetworks

### LINKED ISSUE #818: fix(communication): display changes to ConnectedAPs
**Describe the bug**
After https://github.com/openscd/open-scd/issues/810 was fixed, the communication editor still doesn't reflect changes to `ConnectedAP` elements.

**To Reproduce**
1. Go to 'Communication editor'
2. Click either
  - the 'Add ConnectedAP' button in the top right of any Subnetwork and connect an unconnected Access Point using the wizard, or
  - the `Delete` button of any `ConnectedAP`
3. the change to the APs of the respective IED (new IED added or existing IED deleted) doesn't show up.

**Expected behavior**
The change is reflected in the display of the editor.

**Additional context**
This happens since the changes to plugin loading merged in  #787 .

### PR DESCRIPTION
**Pull request by ca-d**
Closes #818 .

This is a draft because it still requires regression tests, but the fix itself is ready to review.

### CODE REVIEW

**Comment by Flurb** [comment]
Did you test using the Pro Mode XML editor?
When I edit a value of one P element for example in the Pro Mod XML editor of a ConnectedAP, I get the following error:

```
Uncaught TypeError: Cannot read properties of null (reading 'getAttribute')
    at getTypes (p-types.js:5:21)
    at editConnectedApWizard (connectedap.js:156:14)
    at ConnectedAPEditor.openEditWizard (connectedap-editor.js:27:39)
    at ConnectedAPEditor.<anonymous> (connectedap-editor.js:46:32)
    at EventPart.handleEvent (lit-html-fbfed138.js:1059:24)
    at EventPart.__boundHandleEvent (lit-html-fbfed138.js:1025:47)
```

**Review by dlabordus — APPROVED** [review]

File: test/integration/editors/communication/subnetwork-editor-wizarding-editing.test.ts [hunk 1]
```diff
@@ -77,52 +78,65 @@ describe('subnetwork-editor wizarding editing integration', () => {
...
     });
+
     it('changes desc attribute on primary action', async () => {
       descField.value = 'newDesc';
       primaryAction.click();
       await parent.updateComplete;
       expect(doc.querySelector('SubNetwork')?.getAttribute('desc')).to.equal(
         'newDesc'
       );
+      await expect(element).shadowDom.to.equalSnapshot();
```
Comment (JakobVogelsang) [inline test/integration/editors/communication/subnetwork-editor-wizarding-editing.test.ts:99]: Why these snapshots? I don't get what they shall do here other than produce bytes?

**Review by JakobVogelsang — COMMENTED** [review]
In general LGTM. I just don't get what the additional snapshot tests are for?

File: test/integration/editors/communication/subnetwork-editor-wizarding-editing.test.ts (hunk 1 above)
Comment (ca-d) [reply test/integration/editors/communication/subnetwork-editor-wizarding-editing.test.ts:99]: These snapshots are there for testing regressions on the render update issues.
We could replace these by specific DOM property tests in order to see that the changes made in the tests are actually reflected in the user interface. Starting with open-scd-core, I would like to use visual regression tests with screenshots in order to regression test UI bugs like this.

Comment (JakobVogelsang) [reply test/integration/editors/communication/subnetwork-editor-wizarding-editing.test.ts:99]: Ahhh I see that does make sence. Then they are good for now! Thanks for the explanation :)

**Pull request merged**